src/Analysis/Engine/Impl/DependencyResolution/AvailablePackageImports.cs

```diff
@@ -20,13 +20,15 @@
 namespace Microsoft.PythonTools.Analysis.DependencyResolution {
     internal class AvailablePackageImports : IAvailableImports {
         public string Name { get; }
-        public IReadOnlyDictionary<string, string> Modules { get; }
+        public string RootPath { get; }
+        public AvailableModuleImports[] Modules { get; }
         public string[] Packages { get; }
 
-        public AvailablePackageImports(string name, IReadOnlyDictionary<string, string> modules, string[] packages) {
+        public AvailablePackageImports(string name, string rootPath, AvailableModuleImports[] modules, string[] packages) {
```

Comment:
`IAvailableImports[]` or you need a real thing?

Reply:
This collection contains only module imports. I can add an interface for `AvailableModuleImports`.